src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborReaderTests.SkipValue.cs

```diff
@@ -76,17 +72,6 @@ public static void SkipValue_InvalidFormat_ShouldThrowFormatException(string hex
             Assert.Throws<FormatException>(() => reader.SkipValue());
         }
 
-        [Theory]
-        [InlineData("61ff")]
-        [InlineData("62f090")]
-        public static void SkipValue_InvalidUtf8_ShouldDecoderFallbackException(string hexEncoding)
```

Comment:
It's probably still good to test the invalid utf8 specifically, to show it's wrapping the decoder exception.